src/engines/gstenginepipeline.cpp

```diff
@@ -181,9 +181,16 @@ bool GstEnginePipeline::ReplaceDecodeBin(const QUrl& url) {
         spotify_server, "StartPlayback", Qt::QueuedConnection,
         Q_ARG(QString, url.toString()), Q_ARG(quint16, port));
   } else {
+    QByteArray uri;
+    if (url.scheme() == "cdda") {
+      QString str = url.toString();
+      str.remove(str.lastIndexOf(QChar('a')), 1);
```

Comment:
Perhaps it's better to use `QString.leftRef`?
`uri = str.leftRef(str.lastIndexOf(QChar('a'))).toUtf8();`